CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ Our versioning strategy is as follows:
   - includes refactoring of existing Design Library functionality
   - this is a breaking change for applications based on Next.js App Router (beta) template. Please refer to the detailed upgrade guide for further instructions
 * `[core]` Provide information about rendering host as part of code extraction ([#286](https://github.com/Sitecore/content-sdk/pull/286))
+* `[template/next-app-router]` Add component runtime information for preventing mixing client/server operations in Sitecore Pages ([#291](https://github.com/Sitecore/content-sdk/pull/291))
```

Comment:
You can mix components, but the problem that they can be added in "invalid" way, so let's clarify a bit:
```suggestion
* `[template/next-app-router]` Add component runtime information to prevent invalid client/server component operations in Sitecore Pages ([#291](https://github.com/Sitecore/content-sdk/pull/291))
```